content/systems/filecoin_markets/storage_market/storage_client.md

```diff
@@ -2,16 +2,18 @@
 title: Storage Client
 weight: 4
 dashboardWeight: 2
-dashboardState: incomplete
-dashboardAudit: 1
+dashboardState: stable
+dashboardAudit: 0
 dashboardTests: 0
 ---
 
 # Storage Client
 ---
 
-The `StorageClient` is a module that discovers miners, determines their asks, and proposes deals to `StorageProviders`. It also tracks deals as they move through the deal flow. Note that any address registered as a `StorageMarketParticipant` with the `StorageMarketActor` can be used with the `StorageClient`. A single participant can be a client, provider, or both at the same time.
+The `StorageClient` is a module that discovers miners, determines their asks, and proposes deals to `StorageProviders`. It also tracks deals as they move through the deal flow. Note that any address registered as a `StorageMarketParticipant` with the `StorageMarketActor` can be used with the `StorageClient`.
 
-{{< embed src="storage_client.id" lang="go" >}}
+Recall that a single participant can be a `StorageClient`, `StorageProvider`, or both at the same time.
 
-<!-- # Storage Client State Machine -->
+{{<embed src="/modules/go-fil-markets/storagemarket/client.go"  lang="go">}}
+
```

Comment:
you should include the provider as well or move the client to be next to place where the provider is.